node_modules/oae-admin/createtenant/js/createtenant.js

```diff
@@ -45,7 +42,7 @@ define(['jquery', 'underscore', 'oae.core', 'iso3166'], function($, _, oae, iso3
                         'hostname': oae.api.i18n.translate('__MSG__PLEASE_ENTER_A_VALID_HOST_NAME__')
                     },
                     'emailDomain': {
-                        'hostname': oae.api.i18n.translate('__MSG__PLEASE_ENTER_A_VALID_EMAIL_DOMAIN__')
+                        'emailDomain': oae.api.i18n.translate('__MSG__PLEASE_ENTER_A_VALID_EMAIL_DOMAIN__')
```

Comment:
This would change to `'emailDomains'`. And maybe the `PLEASE_ENTER_VALID_EMAIL_DOMAINS` message as well? It could provide instruction that it should be comma-separated list of email domains.